### ISSUE TITLE
[VT-97] [Patch] vitrc.5 man page patch

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#119]
_Ankur Sinha on 2014-06-24T04:05:26Z says:_ 

 *  Fix the vitrc man page

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-06-24T04:05:26Z
Modified: 2014-06-25T19:48:26Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2014-06-25T19:48:26Z says:_ 

 Fixed at d7f928fb. Thanks Ankur.

**Comment by pbeckingham** [comment]
[VT-97_vit-1.2-manpage5.patch.txt](https://github.com/scottkosty/vit/files/1755273/VT-97_vit-1.2-manpage5.patch.txt)
